dashboard/app/models/foorm/form.rb

```diff
@@ -43,12 +43,15 @@ def self.setup
 
       # Let's load the JSON text.
       questions = File.read(path)
+      # if published is not provided, default to true
+      published = questions['published'].nil? ? true : questions['published']
```

Comment:
Is this necessary? Shouldn't the default you set on the `published` column coerce it to true?

Reply:
Yep, it's necessary. If we leave the value as nil it will treat the value as nil, not change it to the default. The default is only used if no value is provided, so we need to check that the value is provided somewhere.